### ISSUE TITLE
RuntimeError: Expected object of scalar type Byte but got scalar type Bool for sequence element

### ISSUE THREAD

**Issue by xiaozhangbiubiubiu** [kenchan0226/keyphrase-generation-rl#5]
when I trained a reinforcement learning model, I met a mistake as below. I don't know which variable is wrong. can you give me some  suggestions?
![error](https://user-images.githubusercontent.com/56903042/67351277-ae928380-f57f-11e9-8b28-82709d30c1c4.png)

**Comment by kenchan0226** [comment]
Hi, did you use Pytorch 0.4 to run our code? may I know which command you use so that I can replicate the problem?

**Comment by xiaozhangbiubiubiu** [comment]
No. I use Pytorch 1.3.0 and the command is "_train.py -data data/kp20k_separated/ -vocab data/kp20k_separated/ -exp_path exp/%s.%s -exp kp20k -epochs 20 -copy_attention -use_target_encoder -train_rl -one2many -one2many_mode 1 -batch_size 32 -separate_present_absent -pretrained_model [path_to_ml_pretrained_model] -max_length 60 -baseline self -reward_type 7 -replace_unk -topk G -seed 9527_" for training a **catseqD-2RF1** model. I change nothing except the data path and model path.
Thanks.

**Comment by kenchan0226** [comment]
Hi, can you use Pytorch 0.4 to run it? I think the reason is that the PyTorch APIs have some changes and the variables returned by the functions are different. You can simply use anaconda to setup another virtual environment with Pytorch 0.4 to run our code. Sorry for any inconvenience, because at the time I wrote the code, PyTorch 0.4 is the newest version.

**Comment by xiaozhangbiubiubiu** [comment]
it can run with Pytorch 0.4 but it can only run on CP possibly because the version of Pytorch and Cuda doesn't match. 
Thank you very much!

**Comment by kenchan0226** [comment]
Hi, I think you can also try Pytorch 1.0.0 I heard that the API of Pytorch 1.0.0 has no major changes compared with Pytorch 0.4.0.
